### PR TITLE
Fix: Populate wild loomies

### PR DESCRIPTION
**Pull request by PedroChaparro**
- [x] Populate the types field as an Strings array. 
- [x] Populate the rarity field as an String.

### CODE REVIEW

**Review by andres123dbh — APPROVED** [review]

**Comment by PedroChaparro** [comment]
Wait for @SilviaPabon approval.

**Review by SilviaPabon — APPROVED** [review]

**Comment by SilviaPabon** [comment]
Thanks!

**Pull request merged**
